tests/metrics/test_toolkit.py

```diff
@@ -83,11 +83,11 @@ def test_clone_metric(self) -> None:
         metric.update(torch.tensor(2.0))
         self.assertDictEqual(clone_metric(metric).state_dict(), metric.state_dict())
         if torch.cuda.is_available():
-            metric.to("cuda")
+            metric.to("cuda:0")
             cloned_metric = clone_metric(metric)
-            self.assertEqual(cloned_metric.device, torch.device("cuda"))
+            self.assertEqual(cloned_metric.device, torch.device("cuda:0"))
             # pyre-ignore[16]: Undefined attribute [16]: `Metric` has no attribute `sum`.
-            self.assertEqual(cloned_metric.sum.device, "cuda")
+            self.assertEqual(cloned_metric.sum.device, torch.device("cuda:0"))
 
```

Comment:
pull out "cuda:0" as its own device variable
```
device = torch.device("cuda:0")
metric.to(device)
self.assertEqual(cloned_metric.device, device)
self.assertEqual(cloned_metric.sum.device, device)
```
?

Reply:
yes. this is more clear~